src/node_mem.h

```diff
@@ -13,8 +13,12 @@ namespace mem {
 // use different struct names. To allow for code re-use,
 // the NgLibMemoryManager template class can be used for both.
 
+struct NgLibMemoryManagerBase {
```

Comment:
standalone PR, also affects nghttp2, right?

Reply:
Not this particular change, but generally yes, a separate PR should be okay